// PB2BP: BP_FILE_NAME=PbGddGlobal
syntax = "proto3";
package idlepb;

import "defines.proto";
import "common.proto";

// ============================================================================

message CommonGlobalConfig {
	float ts_rpc_max_seconds = 1;  // TsRpc 超时时间
}

message GameServicesConfig {
	string listen_ip = 1;
	int32 port = 2;

	string redis_ip = 3;
	int32 redis_port = 4;
	string redis_password = 5;
}

// ============================================================================
